### PR TITLE
LRDOCS-2635 Struts questions for Sergio

### PR DESCRIPTION
**Pull request by mwilliams2014**
Hey @sergiogonzalez I added some questions for you to answer on the tutorial. I just began organizing the information. Before I get too far though, I would like to have those questions answered so I can make sure the steps are clear to the developers. I'll take a look when I get back. thx!

### CODE REVIEW

File: develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown [hunk 1]
```diff
@@ -1,60 +1,99 @@
...
+
+Some of the portlets mapping may have changed, so it’s best to check Liferay's 
+source code to determine the correct mapping.
+
+Depending on the URL it will be a different param:
+
+-   RenderURLs will contain a parameter named `mvcRenderCommandName`
+-   ActionURLs will be contained in the attribute `name` of the taglib
+    or with the parameter `ActionRequest.ACTION_NAME`
+-   RenderURLs will be contained as the attribute `id`.<!-- Is this suppose to be the RenderURLs again, or something else? -->
```
Comment (sergiogonzalez) [inline develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown]: This is supposed to be resourceURLs not renderURLs.

File: develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown [hunk 2]
```diff
@@ -1,60 +1,99 @@
...
+Depending on which method you customized in your StrutsActionWrapper, you will 
+need to use the matching MVCCommand shown below:
 
 -   processAction ->  MVCActionCommand
 -   render -> MVCRenderCommand
 -   serveResource -> MVCResourceCommand
 
-Once you know which MVCCommand you will need to know which is the
-associated mapping that is used by that MVCCommand.
+<!--
```
Comment (sergiogonzalez) [inline develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown]: There's not much code to show here. I'm just trying to explain based on the customizations that the users could have what new java classes they should create new using the MVCCommand.

Take a look at the `ExampleStrutsPortletAction` class from here: https://dev.liferay.com/develop/tutorials/-/knowledge_base/6-2/overriding-and-adding-struts-actions . Depending on which actions were overriden the user will have to use different MVCCommands. In the example https://dev.liferay.com/develop/tutorials/-/knowledge_base/6-2/overriding-and-adding-struts-actions they only override the action and render so in order to migrate to the new pattern they would need to create 2 classes: MVCActionCommand and MVCRenderCommand.

File: develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown [hunk 3]
```diff
@@ -1,60 +1,99 @@
...
+-   ActionURLs will be contained in the attribute `name` of the taglib
+    or with the parameter `ActionRequest.ACTION_NAME`
+-   RenderURLs will be contained as the attribute `id`.<!-- Is this suppose to be the RenderURLs again, or something else? -->
+
+<!--
+
+Can you show the code for this, using the beginning example? In which cases
+would you use each param?
+
+-->
```
Comment (sergiogonzalez) [inline develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown]: You can see examples of how the urls look like here:

https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/mvc-render-command#creating-a-portlet-render-url
https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/mvc-resource-command
https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/mvc-action-command

File: develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown [hunk 4]
```diff
@@ -1,60 +1,99 @@
...
+move onto the tutorials shown below? Is this all done in a `liferay-hook.xml`?
+
+I'm guessing not. So what file are these code changes being made in?
+
+A legacy code example and a finished converted code(using the same example)
+would be good to have here.
+
+Thanks!
+
+-->
```
Comment (sergiogonzalez) [inline develop/tutorials/articles/25-frontend-customizations/xx-converting-struts-action-wrappers-to-mvc-commands.markdown]: All the information about how the code would look like and how to create mvc commands is here https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/overriding-mvc-commands

**Comment by mwilliams2014** [comment]
@sergiogonzalez wordsmithed and updated per your comments. Sending to be published. thx!

**Pull request closed without merging**